lib/checks/lists/only-listitems.js

```diff
@@ -17,6 +17,13 @@ const getHasListItem = (hasListItem, tagName, isListItemRole) => {
 	return hasListItem || (tagName === 'LI' && isListItemRole) || isListItemRole;
 };
 
+const getIsHidden = actualNode => {
+	return (
+		window.getComputedStyle(actualNode, null).getPropertyValue('display') ===
```

Comment:
DRY, worth extracting this method to axe.utils

Reply:
I wasn't aware of the utils, and I see it actually has an is-hidden.js already. Guess I can rewrite using that.

Reply:
Because of the above disparity, I have put in this PR - https://github.com/dequelabs/axe-core/pull/1023 which will give an idea of API methods available.

Reply:
I don't think this should use the existing isHidden method, as that looks up the tree. We don't need that here. Just check for `display:none;` and `visibility:hidden;`.

Reply:
Still worth extracting then?

Reply:
I think so, yes. See my comment above about `aria-hidden="true"`